packages/special-pages/shared/components/Text/Text.js

```diff
@@ -7,11 +7,22 @@ import styles from './Text.module.css'
  * @param {import('preact').JSX.ElementType} [props.as]
  * @param {Omit<keyof styles, "text">} [props.variant]
  * @param {string} [props.className]
+ * @param {boolean} [props.strictSpacing] - Apply Design System letter spacing. Default: true
  * @param {import("preact").ComponentChild} [props.children]
+ * @param {import("preact").JSX.MouseEventHandler<EventTarget>} [props.onClick]
  */
-export function Text ({ as: Comp = 'p', variant, className, children }) {
+export function Text ({ as: Comp = 'p', variant, strictSpacing = true, className, children, onClick }) {
     return (
-        <Comp className={classNames({ [styles[`${variant}`]]: variant }, className)}>
+        <Comp className={classNames({ [styles[`${variant}`]]: variant, [styles.strictSpacing]: strictSpacing }, className)}
+            onClick={
```

Comment:
This is adding an event handler for every use of the Text component. since onClick will only be given to <a> elements, I wouldn't use a generic <Text> for <a> elements. The dance you're having to do with the event handler indicates that this is trying to do too much. 

Why not just have a <Link> element in this same file, and that can re-use styles, but be explicit about handlers.